dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/writer/DumpSchemaJsonProtocol.scala

```diff
@@ -0,0 +1,9 @@
+package org.virtuslab.akkasaferserializer.writer
+
+import org.virtuslab.akkasaferserializer.model.{Field, TypeDefinition}
+import spray.json.{DefaultJsonProtocol, JsonFormat}
+
+trait DumpSchemaJsonProtocol extends DefaultJsonProtocol {
```

Comment:
Nit: there's still a bunch of places where `dump schema` form (with no `event`) is used... pls rename for the sake of Sacred Consistency ™️

```
$ git grep -iE dump.?schema
README.md:### 3. Dump Schema
README.md:libraryDependencies += repo % "sbt-dumpschema" % commit
README.md:enablePlugins(DumpSchemaPlugin)
dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/writer/DumpSchemaJsonProtocol.scala:trait DumpSchemaJsonProtocol extends DefaultJsonProtocol {
dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/writer/SchemaWriter.scala:class SchemaWriter(outputDirectory: File) extends DumpSchemaJsonProtocol {
dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/writer/SchemaWriter.scala:  def this(dumpSchemaOptions: DumpEventSchemaOptions) = {
dump-event-schema-compiler-plugin/src/main/scala/org/virtuslab/akkasaferserializer/writer/SchemaWriter.scala:    this(File(dumpSchemaOptions.outputDir))
dump-event-schema-compiler-plugin/src/test/scala/org/virtuslab/akkasaferserializer/SchemaWriterSpec.scala:import org.virtuslab.akkasaferserializer.writer.{DumpSchemaJsonProtocol, SchemaWriter}
dump-event-schema-compiler-plugin/src/test/scala/org/virtuslab/akkasaferserializer/SchemaWriterSpec.scala:class SchemaWriterSpec extends AnyWordSpecLike with should.Matchers with DumpSchemaJsonProtocol {
sbt-dump-event-schema/src/main/scala/org/virtuslab/akkasaferserializer/DumpEventSchemaKeys.scala:  lazy val dumpEventSchema = taskKey[Unit]("Dump schema")
sbt-dump-event-schema/src/main/scala/org/virtuslab/akkasaferserializer/DumpEventSchemaKeys.scala:  lazy val dumpEventSchemaOutputFile = taskKey[File]("Output file to dump schema to")
sbt-dump-event-schema/src/main/scala/org/virtuslab/akkasaferserializer/DumpEventSchemaKeys.scala:  lazy val dumpEventSchemaOutputFilename = settingKey[String]("Filename to dump schema to")
sbt-dump-event-schema/src/main/scala/org/virtuslab/akkasaferserializer/DumpEventSchemaKeys.scala:  lazy val dumpEventSchemaCompilerPlugin = settingKey[ModuleID]("ModuleId of dump schema plugin")
```

Reply:
Renamed code occurrences, README stuff moved to #46